proj4/src/main/java/org/osgeo/proj4j/datum/Datum.java

```diff
@@ -71,20 +72,22 @@
   public static final Datum NZGD49 = new Datum("nzgd49", 59.47,-5.04,187.44,0.47,-0.1,1.024,-4.5993, Ellipsoid.INTERNATIONAL, "New Zealand Geodetic Datum 1949");
   public static final Datum OSEB36 = new Datum("OSGB36", 446.448,-125.157,542.060,0.1502,0.2470,0.8421,-20.4894, Ellipsoid.AIRY, "Airy 1830");
 
+  static {
+      Datum temp = null;
+      try {
+          temp = new Datum("NAD27", Grid.fromNadGrids("@conus,@alaska,@ntv2_0.gsb,@ntv1_can.dat"), Ellipsoid.CLARKE_1866,"North_American_Datum_1927");
+      } catch (IOException e) {
+          // TODO: Logging
```

Comment:
Should we really be swallowing here and setting NAD27 to NULL? When would this throw?